src/js/mods/modloader.js

```diff
@@ -187,16 +198,25 @@ export class ModLoader {
             this.modLoadQueue.push({
                 modClass,
                 meta,
+                path,
             });
         };
 
         mods.forEach(modCode => {
+            path = modCode.slice(0, modCode.indexOf("\n")).replace("// ", "");
+            if (path.endsWith(".disabled")) {
+                modCode =
+                    `
+                           let $old_shapez = structuredClone(shapez);
+                        ` + modCode;
+            }
             modCode += `
                         if (typeof Mod !== 'undefined') {
                             if (typeof METADATA !== 'object') {
                                 throw new Error("No METADATA variable found");
                             }
                             window.$shapez_registerMod(Mod, METADATA);
```

Comment:
all this just for metadata I feel is overkill; short term I think metadata should be just cached in a separate file, and long term metadata should be statically accessible